index.html

```diff
@@ -1294,8 +1294,17 @@
 
 [=Sites=] sometimes use data in ways that aren't needed for the user's immediate
```

Comment:
This is the one bit of change from https://github.com/w3ctag/privacy-principles/pull/457/files that wasn't obviated by Martin's rewrite. I think it's a better contrast to ancillary than "immediate".

```suggestion
[=Sites=] sometimes use data in ways that aren't needed for the user's primary
```